app.js

```diff
@@ -434,44 +434,62 @@ function completeListDisappearAnimation(elem) {
   elem.classList.add("scaleDown");
 }
 
-function applyMainColors() {
-  root.style.setProperty("--background-color", "white");
-  root.style.setProperty("--box-color", "white");
-  root.style.setProperty("--main-color", "red");
-  root.style.setProperty("--secondary-color", "#01994d");
-  root.style.setProperty("--third-color", "#ddd");
-  root.style.setProperty("--font-color", "black");
+const themes = {
+  main: {
+    '--background-color': 'white',
+    '--box-color': 'white',
+    '--main-color': 'red',
+    '--secondary-color': '#01994d',
+    '--third-color': '#ddd',
+    '--font-color': 'black'
+  },
+  tan: {
+    '--background-color': '#AA8C77',
+    '--box-color': '#e4e3cc',
+    '--main-color': '#157F9E',
+    '--secondary-color': '#332c30',
+    '--third-color': '#E4E3CC',
+    '--font-color': 'black'
+  },
+  green: {
+    "--background-color": "rgb(13, 18, 24)",
+    "--box-color": "white",
+    "--main-color": "rgb(0, 145, 0)",
+    "--secondary-color": "pink",
+    "--third-color": "#eee",
+    "--font-color": "#0D98BA"
+  },
+  mixed: {
+    "--background-color": "#551b17",
+    "--box-color": "#2a4d53",
+    "--main-color": "#e4e3cc",
+    "--secondary-color": "#e4e3cc",
+    "--third-color": "#bb2321",
+    "--font-color": "#e8e8c6"
+  }
+}
+
+function applyTheme(theme = themes.main) {
```

Comment:
The main theme is applied by default in the event that someone forgets to pass in a theme!